components/places/src/db/schema.rs

```diff
@@ -245,9 +245,33 @@ pub fn init(db: &PlacesDb) -> Result<()> {
     Ok(())
 }
 
+/// Helper for upgrade. Intended use:
+///
+/// ```rust,ignore
+/// from = migration(db, from, 2, &[stuff, to, migrate, version2, to, version3])?;
```

Comment:
This makes sense, but threw me off for a sec; I wondered why `migration` checked `cur_version == from` ("isn't 
the point of migration to migrate from an older version to a newer one"...but I misunderstood how it was meant to be used). What do you think of adding something like this?

```suggestion
/// let from = get_current_schema_version()?; // 2
/// from = migration(db, from, 2, &[stuff, to, migrate, version2, to, version3])?;
```